assignment2/src/components/Question1.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+
+function Question1() {
+      const [email, setEmail] = useState("")
+      const [password, setPassword] = useState("")
+      const [details, setDetails] = useState([])
+
+      const addDetail = ()=>{
+            if(email.length === 0){
```

Comment:
Could you please add more validations for email and password fields like pattern and duplicate data. It's just an additional suggestion.